docs/source/data_hub.md

```diff
@@ -0,0 +1,75 @@
+# DataHub
+
+EasyCV summarized various datasets in different fields. At present, we support part of them, and we will gradually support remainings.
+
+**For datasets we already support, please refer to: [prepare_data.md](https://github.com/alibaba/EasyCV/blob/master/docs/source/prepare_data.md).**
+
+- [Classification data](#Classification-data)
+- [Object Detection](#Object-Detection)
+- [Self-Supervised Learning](#Self-Supervised-Learning)
+- [Pose](#Pose)
+- [Image Segmentation](#Image-Segmentation)
```

Comment:
Self-Supervised Learning
Classification data
Object Detection
Image Segmentation -> semantic segmentation or instance segmentation?
Pose

Reply:
follow the upper order

Reply:
Image Segmentation -> for all segmentation  senarios

Reply:
done!